libr/core/cmd_search_rop.c

```diff
@@ -7,6 +7,8 @@
 #include "r_list.h"
 #include "r_types_base.h"
 
+#define COMA_HITS 2
```

Comment:
not a define, do a `const int kCommaHits = 2`

also, what does it mean when you hit 2 commas?